eZ/Bundle/EzPublishCoreBundle/Resources/config/default_settings.yml

```diff
@@ -30,6 +31,10 @@ parameters:
                 template: %ezplatform.default_view_templates.content.line%
                 match: []
         embed:
+            image:
+                template: '%ezplatform.default_view_templates.content.embed_image%'
+                match:
+                    IsEmbedImage: true
```

Comment:
Removed the matcher. Will use content type identifier(s) for now.

After a call with André, distinguishing embed images will have to be done differently in the future.